src/main/groovy/wooga/gradle/version/internal/release/opinion/Strategies.groovy

```diff
@@ -30,10 +31,7 @@ import org.gradle.api.GradleException
 /**
  * Opinionated sample strategies. These can either be used as-is or as an
  * example for others.
- * @see wooga.gradle.version.internal.release.base.VersionStrategy
- * @see SemVerStrategy
- * @see wooga.gradle.version.internal.release.semver.SemVerStrategyState
- * @see PartialSemVerStrategy
+ * @see wooga.gradle.version.internal.release.base.VersionStrategy* @see SemVerStrategy* @see wooga.gradle.version.internal.release.semver.SemVerStrategyState* @see PartialSemVerStrategy
```

Comment:
Be carefull when reformating code that the docs are not folded.

Reply:
My bad.